frontend/src/templates/Field/Table/AddRowFooter.tsx

```diff
@@ -51,19 +42,34 @@ export const AddRowFooter = ({
         type="button"
         onClick={handleAddRow}
       >
-        Add another row
+        {t('features.publicForm.components.table.addAnotherRow')}
         <VisuallyHidden>
-          to the table field. {maxRowAriaDescription}
+          {t('features.publicForm.components.table.addAnotherRowAria')}
         </VisuallyHidden>
       </Button>
 
       <Box>
         <VisuallyHidden aria-live={hasAddedRows ? 'polite' : 'off'} aria-atomic>
-          The table field currently has {maxRowDescription}
+          {t('features.publicForm.components.table.rowAria')}{' '}
+          {Number.isInteger(maxRows)
+            ? t('features.publicForm.components.table.rowMax', {
+                currentRows,
+                count: Number(maxRows),
+              })
+            : t('features.publicForm.components.table.row', {
+                count: currentRows,
+              })}
         </VisuallyHidden>
 
         <Text aria-hidden textStyle="body-2" color="secondary.400">
-          {maxRowDescription}
+          {Number.isInteger(maxRows)
```

Comment:
Would it make sense for L53-61 and L65-72 be a constant variable since both of them should have the same content and is likely to remain that way in the future.